### DISCUSSION TITLE
RecipeMD from Gourmet

### DISCUSSION THREAD

**Discussion by crhode** [RecipeMD/RecipeMD#40] [Show and tell]
Here's a little something I whipped up to convert a Gourmet Recipe Manager database to markdown.

o http://lacusveris.com/recipemd_from_gourmet/recipemd_from_gourmet.zip

**Comment by tstehr** [comment]
Awesome 👍 Would you be okay with me linking to this from [recipemd.org](https://recipemd.org)?

**Comment by crhode** [comment]
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA1

On Tue, 24 Aug 2021 06:05:04 -0700
Tilman Stehr ***@***.***> wrote:

> Awesome — Would you be okay with me linking to this from
> recipemd.org?

That would be fine with me.

Thanks for thinking about recipes as markdown documents.  It had not
occurred to me.

*gourmet* went tubular after I installed Debian "Buster" and made
Python3 default.  I tried to install the Python3 version (not Debian),
but it stumbled over missing Pango structure iterators during printing,
so I had to give it up.  Presumably this will be fixed in the next
Debian release.  I seem to have moved on, though.

- -- 
.. Be Seeing You,
.. Chuck Rhode, Sheboygan, WI, USA
.. Weather:  http://LacusVeris.com/WX
.. 75° — Wind S 14 mph

-----BEGIN PGP SIGNATURE-----

iF0EARECAB0WIQT+MY/5I/LMPSswTbVg2/xipKOWUgUCYSWGbAAKCRBg2/xipKOW
UrbyAJ9pQ/bbGh1DCtB8A8k2k7LGPf8HTwCffaE/6McWYC9cvzXxGIePQcFqjuk=
=33aP
-----END PGP SIGNATURE-----
